### PR TITLE
Fix minor Apply iframe height issue.

### PR DESCRIPTION
**Pull request by TomNUSDS**
In some situations, the Apply iframe is too short and it gets a nested scrollbar. It's most obvious when changing the "zoom" size or if viewing with a mobile css breakpoint.

Fix changes:
- size to use `em` vs `px` for browser zoom levels
- uses "mobile first" breakpoint that's taller for mobile and shorter for desktop.

No good way to test in localhost and staging because the iframe won't load. Attempted to test by applying new sizes to the production site via the debugger.

Rebuilding the css is problematic. Still must delete the style.css file and run the *release build* to get the correct output (not the `serve` or `start).

### CODE REVIEW

File: assets/stylesheets/application/components/_apply.scss [hunk 1]
```diff
@@ -2,7 +2,13 @@
 
 .site-c-apply {
   overflow: auto;
-  width: 100%;
-  min-height: 2500px;
+  min-height: 195em; /* mobile first - ~3120px @font-size=16*/
```
Comment (drew-usds) [inline assets/stylesheets/application/components/_apply.scss]: @TomNUSDS on my localhost (MacOS, current Chrome) the desktop height is working out to ≈ 2560px. The min height listed here was 2500, but currently on the live site the iframe is set to `style="height: 2867px;"` inline on the iframe tag.

We'll probably need to bump the em values here a little more to get things just past 2867px on desktop (and similarly for tablet/phone too).

Comment (drew-usds) [reply assets/stylesheets/application/components/_apply.scss]: _ALSO_ ... oddly ... that inline style is not present on this build of the site, and also isn't something in these current changes. So not sure where we removed that between our most recent deploys.

Comment (TomNUSDS) [reply assets/stylesheets/application/components/_apply.scss]: Oh... removing `width: 100%` is likely a mistake.

I wonder... In the past, I got bit by this issue where the chrome debugger can have this "mapping the source file" feature enabled where any edits you make to test things in the debugger can become actual edits to the source file.

**Review by drew-usds — APPROVED** [review]
New height is showing up for me locally! So should be ok on the live `<iframe>` 👍

**Pull request merged**